IntroToCSharp/Pages/Activities/Variables.razor

```diff
@@ -0,0 +1,148 @@
+@inherits CaptainCoder.Page
+@page "/activities/variables"
+
+<MudText Typo="Typo.h4" Style="font-weight:bold">Variables</MudText>
+<Section Title="Introduction">
+    <MudText>An essential part of a computer program is being about to store data or “remember” it for future use. Each
+        computer has the capacity to “remember” a lot of things. Computers can remember numbers, letters, whole words,
+        and a lot more. However, they can't remember things that the programmer doesn’t explicitly tell them to. So,
+        when we
+        are coding we must use variables to label and store bits and pieces of essential data that we will need later.
+    </MudText>
+</Section>
+<Section Title="The Basics of Variables">
+    <MudText>
+        A variable is a temporary piece of stored data within the computer's memory.
+    </MudText>
+    <Example Title="Home Address">
+        Your personal belongings are stored in your home but in order to find your house you need the home address. The
+        same is true for a variable. It has an address that allows us to access and alter its data.
+    </Example>
+</Section>
+
+<Section Title="What Makes Up A Variable?">
+    <MudList Clickable="true">
+        <MudListItem Icon="@Icons.Material.Filled.Badge" IconColor="Color.Secondary" Text="Name">
+            <NestedList>
+                <MudListItem>
+                    <MudText>This is the “address” that you declare that allows you to access the variable later.
+                    </MudText>
+                </MudListItem>
+            </NestedList>
+        </MudListItem>
+        <MudListItem Icon="@Icons.Material.Filled.Category" IconColor="Color.Warning" Text="Type">
+            <NestedList>
+                <MudListItem>
+                    <MudText>The type refers to the data being stored. Data can be a bunch of different things so it’s
+                        important for the computer to know what type of data it is handling.
```

Comment:
In C#, we must specify the type of data being stored. In the example above, the type of the data being stored is an `int`. Don't worry too much if you're not sure what an `int` is. For now all you need to know is it is used to store a number.

Reply:
Switch type to come before name.